First/src/tutorials/ArrayListDemo.java

```diff
@@ -0,0 +1,36 @@
+package tutorials;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrayListDemo {
+
+	public static void main(String[] args)
+	{
+		List<Integer> arrayList = new ArrayList<Integer>(5);
+		for(int i=0;i<=5;i++)
+		{
+			arrayList.add(i);
+		}
+		
+		//printing elements
+		System.out.println(arrayList);
+		
+		//remove element from index
+		
+		System.out.println(arrayList.remove(3));
+		
+		//printing element one by one
+		
+		for(int i=0; i<arrayList.size();i++)
+		{
+			System.out.println(arrayList.get(i));
+			}
+		}
+ }
+
+
+
+
+
+
```

Comment:
remove extra spaces

Reply:
Hi @prashantkawa 
Done!! Updated code of ArrayList.

Reply:
You just need to update existing file, no need to create new file if asked for code changes

Reply:
I have just changed Classname. No new file added.

Reply:
This spaces are still not removed, if this file is not in use then remove it